src/core/lib/layer-manager.js

```diff
@@ -735,7 +735,7 @@ export default class LayerManager {
    *                        {Object} srcEvent:             native JS Event object
    */
   _onPointerMove(event) {
-    if (event.isDown) {
+    if (event.leftButton || event.rightButton) {
```

Comment:
Just a thought: Seems like this check (exactly which events are involved in dragging) could be delegated to the controller... If you agree maybe add a TODO (and possibly a checkbox in #1422) and we could come back to this?

Reply:
That would make picking only available to apps that use the deck.gl controllers. We do not have a good backward compatibility story for that yet.

Reply:
Could delegate to the controller if available and fall back to this test?

regardless, this was a minor nit.